src/AppInstallerCLITests/Correlation.cpp

```diff
@@ -225,10 +234,10 @@ DataSet GetDataSet_NoNoise()
     dataSet.TestCases = LoadTestData();
 
     // Arbitrary values. We should refine them as the algorithm gets better.
-    dataSet.RequiredTrueMatchRatio = 0.7;
-    dataSet.RequiredFalseMatchRatio = 0.05;
+    dataSet.RequiredTrueMatchRatio = 0.75;
+    dataSet.RequiredFalseMatchRatio = 0;
     dataSet.RequiredTrueMismatchRatio = 0; // There are no expected mismatches in this data set
-    dataSet.RequiredFalseMismatchRatio = 0.3;
+    dataSet.RequiredFalseMismatchRatio = 0.25;

```

Comment:
There are two datasets but they do not necessarily produce the same results even if they use the same data because one adds noise and the other doesn't. For example my original implementation used to have a couple of false matches when there was noise. In this version there are no false positives, but I can imagine a scenario where the true match ratio is lower with noise due to not being able to pick between multiple promising ARP entries (currently we just pick the highest score, but we could be smarter and look at how much higher it is).